app/views/pages/index.html.erb

```diff
@@ -9,24 +9,22 @@
     <% if @logged_out %>
       <h1 class="display-5 fw-bold">Welcome to the Berkeley <br/> Reentry Student Program</h1>
       <div class="d-grid mt-2 gap-2 d-sm-flex justify-content-sm-center pt-2">
-        <button type="button" class="btn btn-secondary btn-lg px-4 gap-3">Login</button>
-        <button type="button" class="btn btn-secondary btn-lg px-4">Sign-up</button>
+      <%= button_to "Login with Google", '/auth/google_oauth2', class: "btn btn-primary btn-lg" %>
       </div>
     <% else %>
       <h1 class="display-5 fw-bold">Welcome to the Berkeley <br/> Reentry Student Program,</h1>
       <h3 class="display-5 name-display mt-3"><%= @name.html_safe %></h3>
       <% if @user_type.include? 'Admin' %>
         <%= link_to "Go to admin dashboard".html_safe, admins_path, class: "btn btn-primary mt-1 btn-lg px-4 gap-3" %>
-      <% end %>
-      <% if @user_type.include? 'Staff' %>
+      <% elsif @user_type.include? 'Staff' %>
         <%# pending %>
-      <% end %>
-      <% if @user_type.include? 'Student' %>
+      <% else %>
```

Comment:
for now, at least before admins & counselors are fully functional, let's assume a user is a student if they have no other type